### PR TITLE
435: Add a retry to the api call in new-filed update script; fix hand…

### PR DESCRIPTION
**Pull request by hannahkates**
…ling of null ulurp numbers

### CODE REVIEW

File: utils/get-video-links.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ const getVideoLinks = projectid => new Promise(async (resolve, reject) => {
     // get the timestamps and video ids associated with this projectid
     const timestamps = await fetch(api_call)
       .then(d => d.json())
-      .then(({ records }) => records.map((record) => {
+      .then(({ records = [] }) => records.map((record) => {
```
Comment (allthesignals) [inline utils/get-video-links.js:46]: Confused why this is being found in the diff? I thought you merged this change in my PR?

File: utils/inject-supporting-document-urls.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ const MILESTONE_TYPES = {
 async function injectSupportDocumentURLs(project) {
   // extract trimmed ULURP number
   const ulurpNumbers = project.actions
+    .filter(({ dcp_ulurpnumber }) => dcp_ulurpnumber) // filter out nulls
```
Comment (allthesignals) [inline utils/inject-supporting-document-urls.js:18]: Confused why this is being found in the diff? I thought you merged this change in my PR?

**Review by allthesignals — APPROVED** [review]
Ignore my comments, didn't realize this was a merge from develop into master :P

**Comment by ghost** [comment]
did you squash develop for this merge? i think we want to leave the separate commits separate in master, &  diff should be 2 commits, not 1

**Comment by hannahkates** [comment]
I did squash, but that make sense that it wasn't needed. Tests are failing and I don't know why. I pulled the latest from both master and develop before rebasing.

**Comment by ghost** [comment]
Tests aren't failing, the deployment fails because you're trying to push a new commit lineage that diverges from what exists in remote for dokku staging due to the squash. I would try this again without the squash as that's what breaking things 

```
Warning: Permanently added the ECDSA host key for IP address '45.55.119.18' to the list of known hosts.

To dokku@zap-api.planninglabs.nyc:zap-api-staging
 ! [rejected]        develop -> master (fetch first)
error: failed to push some refs to 'dokku@zap-api.planninglabs.nyc:zap-api-staging'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
Exited with code 1
```

**Pull request merged**